app/src/main/java/sergio/splashbaseviewer/mvp/presenter/SplashPresenter.java

```diff
@@ -0,0 +1,42 @@
+package sergio.splashbaseviewer.mvp.presenter;
+
+import android.support.annotation.NonNull;
+
+import com.squareup.otto.Subscribe;
+
+import java.util.List;
+
+import sergio.splashbaseviewer.mvp.model.SplashModel;
+import sergio.splashbaseviewer.mvp.view.SplashView;
+import sergio.splashbaseviewer.utils.models.SplashImage;
+
+/**
+ * @author s.ruiz
+ */
+
+public class SplashPresenter implements SplashModel.SplashServiceConsumer{
+    @NonNull
+    private SplashView view;
+    @NonNull
+    private SplashModel model;
+
+    public SplashPresenter(SplashView view) {
+        this.model = new SplashModel(this);
```

Comment:
The objects model and view are Instantiated in the Activity. Never in the presenter. Remmember this. :)